rlclientlib/logger/http_transport_client.h

```diff
@@ -66,15 +71,23 @@ class http_transport_client : public i_sender
     int join();
 
   private:
-    pplx::task<web::http::status_code> send_request(size_t try_count);
+    // repeat request until success or _max_retries attempted
+    // returns the http::status_code of the final attempt.
+    pplx::task<web::http::status_code> send_request();
+
+    // repeat request until success or _max_retries attempted
+    // returns the http_reponse of the final attempt.
+    pplx::task<http_response> send_request_with_retries(size_t try_count);
 
     i_http_client* _client;
     http_headers _headers;
     buffer _post_data;
 
     pplx::task<web::http::status_code> _task;
 
-    size_t _max_retries = 1;
+    std::chrono::time_point<std::chrono::system_clock> _start_time = std::chrono::system_clock::now();
```

Comment:
shouldn't initialization to `system_clock::now()` be done in the constructor instead of right here?

Reply:
You are correct.  I removed the default assign in the latest diff.

Reply:
Did you forget to commit/push something? I'm still seeing the call to `now()` both right here and again in the constructor. This line should be something like
```
std::chrono::time_point<std::chrono::system_clock> _start_time = 0
```